test/cmdlineTests/dup_opt_peephole/output

```diff
@@ -6,12 +6,12 @@ EVM assembly:
   callvalue
   dup1
   iszero
-  tag_1
+  tag_2
```

Comment:
Actually, I don't really see where the increment in all tags by one is coming from here yet. Might be harmless, but probably worth investigating how this happened.

Reply:
I still don't really understand where these are coming from...

Reply:
It's rather spooky actually.. @chriseth do you by any chance see anything that should affect this in the PR here?

Reply:
If EVMCodeTransform delays return variable init, it might be that a `pushtag` / `tag` can be optimized away by the legacy optimizer. You could take a look at the diff of the assemby before optimization.

Reply:
This test, though, is just compiled with ``--asm`` and is a contract with only a fallback and some inline assembly... Ah right! Duh - I was actually wondering why this hits the EVMCodeTransform at all, but of course, the inline assembly! Ok, I'll have a deeper look.